lib/MultiModule.js

```diff
@@ -15,10 +15,16 @@ class MultiModule extends Module {
 		// Info from Factory
 		this.dependencies = dependencies;
 		this.name = name;
+		this._identifier = null;
 	}
 
 	identifier() {
-		return `multi ${this.dependencies.map(d => d.request).join(" ")}`;
+		if (this._identifier) return this._identifier;
+
+		this._identifier = `multi ${this.dependencies
```

Comment:
Could you compute this within the constructor?

Reply:
Why do you want to save it inside constructor instead of lazy compute?

Reply:
We did it this way on other module classes. Moreover, `identifier` is computed at least once. There is no benefit to compute it lazily.

Reply:
I changed it in `MultiModule.js`